client/webserver/webserver.go

```diff
@@ -342,7 +342,7 @@ func New(cfg *Config) (*WebServer, error) {
 			apiAuth.Post("/walletsettings", s.apiWalletSettings)
 			apiAuth.Post("/orders", s.apiOrders)
 			apiAuth.Post("/order", s.apiOrder)
-			apiAuth.Post("/withdraw", s.apiWithdraw)
+			apiAuth.Post("/send", s.apiSend)
```

Comment:
I don't think there are any other http api users besides our browser frontend, but if there are, they'll get 404.  It's fine I guess, but generally I'd keep the `/withdraw` route and have it wrap some code shared with `apiSend`.  Then we'd mark it deprecated and make that clear in the release notes.